Makefile

```diff
@@ -46,7 +46,7 @@ lint:
 
 build:
 	@echo "+ $@"
-	@go build -v ${GO_LDFLAGS} ./...
+	@go build -v  -gcflags '-N -l' ${GO_LDFLAGS} ./...
```

Comment:
This builds in debug mode, so gdb is semi-useful.  I'll get rid of it before the final merge.